packages/ast-position/test/ast-position-spec.js

```diff
@@ -0,0 +1,69 @@
+const { getAstNodeInPosition } = require("../lib/api");
+const { buildAst } = require("@xml-tools/ast");
+const { expect } = require("chai");
+const { TextDocument } = require("vscode-languageserver");
+const { parse } = require("@xml-tools/parser");
+
+describe("AST Position visitor", () => {
```

Comment:
Also add a check in all the positive tests that the returned value is defined (`exists` assertion)